hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/HandleWorkflow.java

```diff
@@ -85,6 +86,7 @@ public class HandleWorkflow {
     private final ConfigProvider configProvider;
     private final InstantSource instantSource;
     private final HederaRecordCache recordCache;
+    private final ConsensusTimeHook consensusTimeHook;
```

Comment:
Given this is staking-specific, should it be named `stakingPeriodTimeHook`?

Reply:
This is staking-specific for now, but there are non-staking related things that will go in here as well

Reply:
Done